csrankings-c.csv

```diff
@@ -473,6 +473,7 @@ Chengwen Luo 0001,Shenzhen University,https://luochengwen.bitbucket.io,DPW5Pj8AA
 Chengxiang Zhai,Univ. of Illinois at Urbana-Champaign,http://czhai.cs.illinois.edu,YU-baPIAAAAJ
 Chengyu Song,Univ. of California - Riverside,https://www.cs.ucr.edu/~csong,EoypoXAAAAAJ
 Chengzheng Sun,Nanyang Technological University,http://www.ntu.edu.sg/home/czsun,eBpYUn4AAAAJ
+Chenhao Lin,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/zh/web/linchenhao/cv,YK0G990AAAAJ

```

Comment:
Research fellow; not eligible for inclusion - please remove.

Reply:
Removed